src/screens/Events/EventsMap.js

```diff
@@ -112,9 +112,9 @@ class EventsMap extends Component {
     const {
       props: { events: fetchedEvents }
     } = this;
-    const previousEventIds = prevProps.events.map(event => event.id);
+    const previousEventIds = prevProps.events.map(event => event._id);
```

Comment:
Why did we transition to `_id` here?

Reply:
The underscore is actually a default of the server, and while we had a server-side i18n library hooked up I took out the underscore on the id to keep that library from getting confused. Now that the library is taken out, though, none of that is necessary, so I changed it back.